client/src/app/primitives/TabLinks.js

```diff
@@ -33,7 +33,8 @@ const TABS_OPTS = {
   }
 };
 
-const SMALL_TAB_WIDTH = 45;
+const SMALL_TAB_WIDTH = 90;
+const SMALLER_TAB_WIDTH = 45;
```

Comment:
(minor) `small` and `smaller` are confusing names I feel. Maybe it would simply help to add a comment here (or find better names, but I don't have any at hand)

Reply:
I added a comment accordingly to achieve more clarity (couldn't find better names as well), https://github.com/camunda/camunda-modeler/pull/2655/commits/898f1ea4d67df6cf90d29716155c52c6989b7177.

Reply:
:+1: